core/components/atoms/_overlay/overlay.js

```diff
@@ -13,6 +14,12 @@ const keyCodes = {
   escape: 27
 }
 
+export const overlayContentSizes = {
```

Comment:
// TODO for laters: tokenise these

Reply:
Agree, we're going to wait till we have the 2nd version of tokens.